client/windows/CreaturePurchaseCard.h

```diff
@@ -17,22 +17,23 @@ class CButton;
 class CreatureCostBox;
 class QuickRecruitmentWindow;
 
-class CreaturePurhaseCard : public CIntObject
+class CreaturePurchaseCard : public CIntObject
 {
 public:
 	const CCreature * creatureOnTheCard;
 	std::shared_ptr<CSlider> slider;
 	QuickRecruitmentWindow * parent;
 	int maxAmount;
 	void sliderMoved(int to);
-	CreaturePurhaseCard(const CCreature * creature, Point position, int creaturesMaxAmount, QuickRecruitmentWindow * parents);
-
+	CreaturePurchaseCard(std::vector<CreatureID> creaturesID, Point position, int creaturesMaxAmount, QuickRecruitmentWindow * parents);
```

Comment:
this need to be `const std::vector<CreatureID> &`

Reply:
fixed